src/foam/mlang/mlangJava.js

```diff
@@ -492,6 +492,19 @@ foam.CLASS({
   ]
 });
 
+foam.CLASS({
```

Comment:
Weren't we going to get rid of the DateConstant and just make the wrapped PreparedStatement smarter?

Reply:
You mean: add a if statement inside the IndexPreparedStatement->setObject(). If it is Date type, use delegate.setObject(obj, java.sql.types.TimeStamp)?

Reply:
Yes

Reply:
That is make sense. Since only Postgres has problem on the timeStamp, it dose not mean that other databases have same issues.